Dockerfile

```diff
@@ -1,67 +1,24 @@
-####### TEST BUILD #######
-ARG UBUNTU_DISTRO
-
-FROM ubuntu:$UBUNTU_DISTRO as test-build
+ARG ROS_DISTRO
 
-ARG UBUNTU_DISTRO
-ARG ROS_VERSION
+FROM ros:${ROS_DISTRO}-ros-core
 
-# setup timezone
-RUN echo 'Etc/UTC' > /etc/timezone && \
-    ln -s /usr/share/zoneinfo/Etc/UTC /etc/localtime && \
+RUN export DEBIAN_FRONTEND=noninteractive; \
     apt-get update && \
-    apt-get install -q -y --no-install-recommends tzdata && \
-    rm -rf /var/lib/apt/lists/*
-
-# install packages
-RUN apt-get update && apt-get install -q -y --no-install-recommends \
-    dirmngr \
-    gnupg2 \
+    apt-get install -y --no-install-recommends \
+    bash-completion \
+    pkg-config \
+    git \
+    vim \
     && rm -rf /var/lib/apt/lists/*
 
-# setup keys
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
-
-# setup sources.list
-RUN echo "deb http://packages.ros.org/ros$ROS_VERSION/ubuntu $UBUNTU_DISTRO main" > /etc/apt/sources.list.d/ros${ROS_VERSION}-latest.list
-
-# setup environment
-ENV LANG C.UTF-8
-ENV LC_ALL C.UTF-8
-
-# install bootstrap tools
-RUN apt-get update && apt-get install --no-install-recommends -y \
-    build-essential \
-    git \
+# install ros tools
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
     python3-colcon-mixin \
-    python3-rosdep \
```

Comment:
> Why did you remove these packages?

the ros_base already comes with the [proper rosdep version](https://github.com/osrf/docker_images/blob/b37c2bedbdd06ee0d12aa6581359ad23900aa335/ros/noetic/ubuntu/focal/ros-base/Dockerfile#L8). Still, I must change the base image to ros:${ROS_DISTRO}-ros-base (I've forgot about this change).

I think is better letting the base_image install the right rosdep version rather than passing it as an argument.